app/core/services/stripe.coffee

```diff
@@ -1,7 +1,9 @@
 publishableKey = if application.isProduction() then 'pk_live_27jQZozjDGN1HSUTnSuM578g' else 'pk_test_zG5UwVu6Ww8YhtE9ZYh0JO6a'
 
 if me.isAnonymous()
-  module.exports = {}
+  module.exports = {
+    openAsync: _.noop
```

Comment:
I added this so it could be spied on in [SubscribeModal tests](https://github.com/codecombat/codecombat/blob/sub-ab-test/test/app/views/core/SubscribeModal.spec.coffee#L70-70), but that's not clear here. How about I just add a comment clarifying that? I could also just make it a function that returns a rejected Promise so that if this gets called while the user is anonymous, an error occurs.